src/lib/mdns/Discovery_ImplPlatform.cpp

```diff
@@ -478,7 +478,12 @@ void DiscoveryImplPlatform::HandleNodeIdResolve(void * context, MdnsService * re
     nodeData.mAddress     = result->mAddress.ValueOr({});
     nodeData.mPort        = result->mPort;
 
-    ChipLogProgress(Discovery, "Node ID resolved for 0x" ChipLogFormatX64, ChipLogValueX64(nodeData.mPeerId.GetNodeId()));
+#if CHIP_PROGRESS_LOGGING
+    char addrBuffer[INET6_ADDRSTRLEN + 1];
```

Comment:
Could you change this to kMaxIPAddressStringLength? It will sort itself out for the different platforms that way.

Reply:
Good catch, will do.  Hadn't noticed `kMaxIPAddressStringLength` somehow.